01_fibonacci/fib.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fibonacci(10)
+}
+
+func fibonacci(n int) {
+
+	var sum int
+	var prev, cur int = 0, 1
```

Comment:
Nice one-liner. You should use `:=`, though.

Reply:
Yes, corrected